File: Kattis/hakkari/hakkari.cpp
#include <iostream>
#include <vector>
#include <utility>
#include <string>

int main() {
    int n, m;
    std::cin >> n >> m;
    int numBombs = 0;
    std::vector<std::pair<int, int>> bombLocations;
    bombLocations.reserve(n * m);

    for (int i = 0; i < n; i++) {
        std::string row;
        std::cin >> row;

        for (int j = 0; j < m; j++) {
            if (row[j] == '*') {
                numBombs++;
                bombLocations.push_back({i + 1, j + 1});
            }
        }
    }

    std::cout << numBombs << "\n";

    for (std::pair location : bombLocations) {
        std::cout << location.first << " " << location.second << "\n";
    }
    return 0;
}

File: Kattis/hakkari/hakkari.py
n, m = map(int, input().split())
num_bombs = 0
bomb_locations = []

for i in range(n):
    row = input()

    for j in range(m):
        if row[j] == '*':
            num_bombs += 1
            bomb_locations.append((i + 1, j + 1))

print(num_bombs)

for location in bomb_locations:
    print(f"{location[0]} {location[1]}")